_posts/2018-11-11-kanban.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "My Kanban Journey"
+category: puppet
+tags: puppet agile kanban
+---
+
+***Disclaimer: This post was written as an employee of Puppet, Inc., but does not constitute opinions of the company.***
+
+Last year I was part of the effort to get the [Puppet Development Kit](https://puppet.com/blog/develop-modules-faster-new-puppet-development-kit) built and shipped. Since I'm very interested in team effectiveness and agility, at one point I took a day to re-organize our kanban boards to streamline the info exposed and reduce the friction when using them. Only after leaving the team, I realised [the value of having a low-friction JIRA configuration](https://twitter.com/nigelkersten/status/1024675669429518336):
```

Comment:
On the line starting with "Since I'm very interested in team efficiencies" perhaps introduce you were at that point in time.  You jump right into making the process more efficient without talking about the state of things at the time.  It was quite a journey from the brainstorming trello board, to the basic Jira implementation  that we had to the complete refactoring of our Jira boards that you did.

Reply:
rereading it, it also sounds pretentious as feck. changed it.